install_pim/manual/crontab_tasks.rst.inc

```diff
@@ -25,7 +26,8 @@ In the following example, these lines will run the versioning calculation at 5am
 
     # m  h  dom  mon  dow  command
     0 5  *    *    *    php /path/to/installation/pim-community-standard/bin/console pim:versioning:refresh --env=prod > /path/to/installation/pim-community-standard/var/logs/refresh_versioning.log 2>&1
-    0 22  *    *    *    php /path/to/installation/pim-community-standard/bin/console pim:volume:aggregate --env=prod > /path/to/installation/pim-community-standard/var/logs/volume_aggregate.log 2>&1
+    1 *  *    *    *    php /path/to/installation/pim-community-standard/bin/console akeneo:connectivity-audit:update-data --env=prod > /path/to/installation/pim-community-standard/var/logs/refresh_audit_data.log 2>&1
```

Comment:
`In the following example, these lines will run the versioning calculation at 5am and the aggregate volume at 10pm every day:`

not up to date. (Same comment as below, maybe useless)